docs/ops.html

```diff
@@ -4316,7 +4316,26 @@ <h4 class="anchor-heading"><a id="tiered_storage_config_ex" class="anchor-link">
 
 <pre><code class="language-bash">$ bin/kafka-console-consumer.sh --topic tieredTopic --from-beginning --max-messages 1 --bootstrap-server localhost:9092 --property print.offset=true</code></pre>
 
-<p>Please note, if you want to disable tiered storage at the cluster level, you should delete the tiered storage enabled topics explicitly.
+<p>If you want to disable tiered storage at the topic level to let the remote logs become read-only and no more local logs copied to the remote storage,
```

Comment:
Please clarify that the disablement feature is available only with KRaft.

Reply:
Oh, good catch! Updated.